django/core/handlers/wsgi.py

```diff
@@ -130,7 +133,13 @@ def __call__(self, environ, start_response):
         set_script_prefix(get_script_name(environ))
         signals.request_started.send(sender=self.__class__, environ=environ)
         request = self.request_class(environ)
-        response = self.get_response(request)
+
+        # Get the response, either using async get_response or standard.
+        if asyncio.iscoroutinefunction(self.get_response):
+            response = async_to_sync(self.get_response)(request)
```

Comment:
Is this block correct? 

`self.get_response()` is `BaseHandler.get_response()`, which in contrast to `get_response_async()` is **not** `async def`, so will this branch ever evaluate as `True`? 

The contrasting point in ASGIHandler just does `await self.get_response_async(request)`, allowing the branching inside `_get_response_async()` once the callback is resolved. 

This line isn't hit by the test suite.

Reply:
This is a relic of the previous attempt - looks like I got the one in `ASGIHandler` but not this. Pushed a commit to remove it.